docs/docs/sample-scripts/teams/export-all-channels-teams.md

```diff
@@ -0,0 +1,57 @@
+# Export all channels in Microsoft Teams teams in the tenant
+
+Author: [Sudharsan Kesavanarayanan](https://twitter.com/sudharsank)
+
+Export all the channels from Microsoft Team in a CSV.
+
+```powershell tab="PowerShell"
+function Get-Channels(
+    [Parameter(Mandatory = $false)][string] $teamID,
+    [Parameter(Mandatory = $false)][string] $teamName 
+) {
+    if(!$teamID -and !$teamName) {
+        Write-Error "Either 'Team ID' or 'Team Name' is required"
```

Comment:
Let's use the property names (`teamID` and `teamName`) to make it clearer which values we expect